library/cwm/src/modules/CWM.rb

```diff
@@ -742,7 +742,6 @@ def PrepareDialog(dialog, widgets)
         [widget_key, w]
       end.to_h
       t = ProcessTerm(dialog, m)
-      t = deep_copy(t)
```

Comment:
please document it in method documentation that it is not deep copy, so can modify part of passed dialog.